core/src/main/groovy/noe/common/utils/Platform.groovy

```diff
@@ -120,6 +120,10 @@ class Platform {
     return (isRHEL() && (osVersion ==~ /.*el7.*/)) || forceRhel7()
   }
 
+  boolean isRHEL8() {
+    return (isRHEL() && (osVersion ==~ /.*el8.*/))
```

Comment:
Please add a method `forceRhel8`, which will work the same as `forceRhel7` (for docker purposes)

Reply:
updated